nomenclature/testing.py

```diff
@@ -50,4 +51,11 @@ def assert_valid_structure(path: Path):
     nomenclature.DataStructureDefinition(path / "definitions")
 
 
+def assert_valid_mappings(path: Path):
```

Comment:
With this implementation, the parsing of the DataStructureDefinition happens twice - that seems not very efficient. In my biew, the "structure" is both the definitions and the mappings, so this could be just one testing function.

Reply:
Ah right, ok, I'll update that.

Reply:
Done.